src/System.Linq/src/System/Linq/First.cs

```diff
@@ -34,17 +32,11 @@ public static TSource First<TSource>(this IEnumerable<TSource> source, Func<TSou
             return first;
         }
 
-        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source)
-        {
-            bool found;
-            return source.TryGetFirst(out found);
-        }
+        public static TSource FirstOrDefault<TSource>(this IEnumerable<TSource> source) =>
+            source.TryGetFirst(out bool found);
```

Comment:
Actually, I just remembered that this is now actually encouraged by C#, so I shall when I come back to this, this evening.